x/incentive/keeper/suite_test.go

```diff
@@ -0,0 +1,148 @@
+package keeper_test
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	sdkmath "cosmossdk.io/math"
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
+	"github.com/stretchr/testify/suite"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
+	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
+
+	umeeapp "github.com/umee-network/umee/v4/app"
+	appparams "github.com/umee-network/umee/v4/app/params"
+	"github.com/umee-network/umee/v4/x/incentive"
+	"github.com/umee-network/umee/v4/x/incentive/keeper"
+	incentivemodule "github.com/umee-network/umee/v4/x/incentive/module"
+
+	"github.com/umee-network/umee/v4/x/leverage/fixtures"
+)
+
+const (
+	umeeDenom = appparams.BondDenom
+	atomDenom = fixtures.AtomDenom
+	daiDenom  = fixtures.DaiDenom
+)
+
+type IntegrationTestSuite struct {
+	suite.Suite
```

Comment:
We should not use `testify.Suite` (we discussed that few times, and migrated some old tests). Basically, we can't run tests in parallel.

Reply:
Tests have been refactored to unit tests